src/Nox.Types/Types/DateTime/DateTime.cs

```diff
@@ -0,0 +1,176 @@
+    {
+        return dateTime1.Value - dateTime2.Value;
+    }
+
+    public static bool operator >(DateTime dateTime1, DateTime dateTime2)
+    {
+        return dateTime1.Value > dateTime2.Value;
+    }
+
+    public static bool operator <(DateTime dateTime1, DateTime dateTime2)
+    {
+        return dateTime1.Value < dateTime2.Value;
+    }
+
+    public static bool operator <=(DateTime dateTime1, DateTime dateTime2)
+    {
+        return dateTime1.Value <= dateTime2.Value;
+    }
+
+    public static bool operator >=(DateTime dateTime1, DateTime dateTime2)
+    {
+        return dateTime1.Value >= dateTime2.Value;
+    }
+
+    /// <summary>
+    /// Returns a string representation of the custom date and time using <see cref="CultureInfo.InvariantCulture"/>.
+    /// </summary>
+    public override string ToString()
+    {
+        return Value.ToString(CultureInfo.InvariantCulture);
+    }
+
+    /// <summary>
+    /// Returns a string representation of the custom date and time using the provided <paramref name="cultureInfo"/>.
+    /// </summary>
+    /// <param name="cultureInfo"></param>
+    public string ToString(CultureInfo cultureInfo)
+    {
+        return Value.ToString(cultureInfo);
+    }
+
+    /// <summary>
+    /// Returns a string representation of the custom date and time using the provided <paramref name="cultureInfo"/> and <paramref name="format"/>.
+    /// </summary>
+    /// <param name="format"></param>
+    /// <param name="cultureInfo"></param>
+    public string ToString(string format, CultureInfo cultureInfo)
+    {
+        return Value.ToString(format, cultureInfo);
+    }
+
+    /// <summary>
+    /// Returns a string representation of the custom date and time using <see cref="CultureInfo.InvariantCulture"/> and the provided <paramref name="format"/>.
+    /// </summary>
+    /// <param name="format"></param>
+    public string ToString(string format)
+    => Value.ToString(format, CultureInfo.InvariantCulture);
+
+    /// <summary>
+    /// Returns a string representation of the custom date and time using <see cref="IFormatProvider"/>.
+    /// </summary>
+    /// <param name="formatProvider"></param>
+    public string ToString(IFormatProvider formatProvider)
+    {
+        return Value.ToString(formatProvider);
+    }
+
+    /// <summary>
+    /// Returns a string representation of the custom date and time using <see cref="IFormatProvider"/> and wanted format.
+    /// </summary>
+    /// <param name="format"></param>
+    /// <param name="formatProvider"></param>
+    /// <returns></returns>
+    public string ToString(string format, IFormatProvider formatProvider)
+    {
+        return Value.ToString(format, formatProvider);
+    }
+
+    /// <summary>
+    /// Adds sent Timespan on <see cref="DateTime.Value"/>
+    /// </summary>
+    /// <param name="ts"></param>
+    public void Add(TimeSpan ts)
+    {
+        Value = Value.Add(ts);
```

Comment:
Nox.Types are immutable, needs to create a new Instance like From, can be marked as static

Reply:
Add removed. It was added in case DateTimeRange uses Nox.Types.DateTime, but now it is redundant since we have operator +